docs/changelog.md

```diff
@@ -0,0 +1,88 @@
+---
+id: Changelog
+toc_min_heading_level: 2
+toc_max_heading_level: 2
+---
+
+# Changelog
+
+## October 6, 2023
+
+### Breaking Changes (Code Incompatibility)
+
+❗[responsive-pub/134](https://github.com/responsivedev/responsive-pub/pull/134) 
+and [responsive-pub/135](https://github.com/responsivedev/responsive-pub/pull/135)
+changed the way a new instance of `ResponsiveKafkaStreams` is created. These
+changes apply to `dev.responsive.kafka-client` versions `0.12.0` and above.
+
+Please modify your code to use the constructor place of the static factory 
+method:
+
+```diff showLineNumbers
+-  KafkaStreams streams = ResponsiveKafkaStreams.create(...);
++  KafkaStreams streams = new ResponsiveKafkaStreams(...);
+```
+
+It also removed the timestamped verisons of `StoreSuppliers` since all Responsive
+stores are timestamped. You can apply all of these changes with the following
+`sed` commands:
```

Comment:
```suggestion
It also removed the timestamped versions of the `StoreSuppliers` since all Responsive
store implementations are compatible with timestamps. To create a timestamped store, 
you now only need to specify it in one place: the `StoreBuilder`. For a timestamped
KeyValueStore you will still be using the `ResponsiveStores#timestampedKeyValueStoreBuilder` 
API, but can now pass in any kind of `KeyValueBytesStoreSupplier` provided by `ResponsiveStores`.

You can apply all of these changes to your application code with the following `sed` commands:
```